webapp/channels/src/components/admin_console/permission_schemes_settings/__snapshots__/permission_description.test.tsx.snap

```diff
@@ -69,7 +69,7 @@ exports[`components/admin_console/permission_schemes_settings/permission_descrip
             <span>
               Inherited from 
               <a
-                key=".$.1"
+                key="1/.1"
```

Comment:
This and other related to formatMessage are due to react-intl upgrade from 6.6.2 to 7.1.14. Note that the workspace packages declare "react-intl": "*", which defers to the lockfile. When I regenerated package-lock.json for node 24, npm resolved * to the latest version (7.1.14 at the time). I now pinned the version in webapp/package.json to make future upgrades intentional and observable but remain * on packages so that same version are used by all.

Reply:
Negative on conflicts to my knowledge. The overall diffs look good and there are no categorical conflicts.